recipe/meta.yaml

```diff
@@ -17,20 +18,20 @@ build:
     - vc10  # [win and py34]
     - vc14  # [win and py>34]
   skip: True  # [py<34 or win]
-  script: python setup.py install --single-version-externally-managed --record record.txt
 
 requirements:
   build:
     - python
     - setuptools
     - toolchain
+    - rapidjson =={{ rapidjson_version }}
 
   run:
     - python
 
 test:
   imports:
-    - rapidjson 
+    - rapidjson
 
 about:
   home: https://github.com/kenrobbins/python-rapidjson
```

Comment:
Maybe it would be better to point to the new _python-rapidjson_ organization URL here?